### PR TITLE
correct code-format name and remove mkdir

### PR DESCRIPTION
**Pull request by autoih**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ us**
 ## Development Tips
 Try these useful commands below:
 
-* Format code automatically: `bash tools/run_docker.sh -c 'make code-format'`
```
Comment (gabrieldemarmiesse) [inline CONTRIBUTING.md]: It's already done in #1028

File: tools/pre-commit.sh [hunk 2]
```diff
@@ -2,6 +2,5 @@
 
 set -e
 
-export DOCKER_BUILDKIT=1
```
Comment (gabrieldemarmiesse) [inline tools/pre-commit.sh]: This was done to avoid long lines. It makes it more readable.

File: tools/docker/Dockerfile_formatting [hunk 3]
```diff
@@ -8,7 +8,6 @@ RUN bash buildifier.sh
 COPY tools/ci_build/install/clang-format.sh ./clang-format.sh
 RUN bash clang-format.sh
 
-RUN mkdir /addons
```
Comment (gabrieldemarmiesse) [inline tools/docker/Dockerfile_formatting]: I didn't know doing WORKDIR created a directory! :) Good to know, it'll save us some typing in the future.

File: CONTRIBUTING.md (hunk 1 above)
Comment (autoih) [reply CONTRIBUTING.md]: Thanks @gabrieldemarmiesse.

File: tools/pre-commit.sh (hunk 2 above)
Comment (autoih) [reply tools/pre-commit.sh]: I see. Just wanna be consistent wiht `run_sanity_check.sh`, that's why I remove it. I'll add it back. Thanks @gabrieldemarmiesse.

**Pull request merged**